client/data/emails/use-remove-email-forward-mutation.ts

```diff
@@ -24,7 +24,7 @@ export function useRemoveEmailForwardMutation(
 	const selectedSiteId = useSelector( getSelectedSiteId );
 
 	const useGetEmailAccountsQueryKey = getEmailAccountsQueryKey( selectedSiteId, domainName );
-	const useGetEmailDomainsQueryKey = getEmailDomainsQueryKey( selectedSiteId );
+	const useGetDomainsQueryKey = getEmailDomainsQueryKey( selectedSiteId );
```

Comment:
This is a bit confusing for readability as hooks are often prefixed with `use...` by convention. Should we take the opportunity to name the variable like:

`const emailDomainsQueryKey = getEmailDomainsQueryKey( selectedSiteId );`

Reply:
Hmm, I agree! I guess my thinking was that I would tack on `QueryKey` at the end of the hook name, but it can indeed appear a bit confusing now that I return to it. Let's go with your suggestion.